internal/table/table.go

```diff
@@ -1,9 +1,11 @@
 package table
```

Comment:
This should not live in the table package. It's use won't be exclusive to tables.

Reply:
I've moved it to the `colorize` package as it's dealing with the tags that that package defines. Let me know if there is a better spot